### DISCUSSION TITLE
add farmers delight compat

### DISCUSSION THREAD

**Discussion by TheBearodactyl** [P3pp3rF1y/SophisticatedBackpacks#1164] [Feature / Idea Archive]
as it is rn, farmers delight foods don't work with the adv. feeding/feeding upgrades

**Comment by P3pp3rF1y** [comment]
I am pretty sure they do, could you be specific about what food doesn't work?

**Reply by TheBearodactyl** [reply]
i've tested most of the food from it and none of them let me put them in the filter slot
